src/main/java/com/dsec/backend/service/UserService.java

```diff
@@ -20,7 +20,7 @@ public interface UserService {
 
 	UserEntity updateUser(long id, UserDTO userDTO, Jwt jwt);
 
-	UserEntity deleteUser(long id, Jwt jwt);
+	UserEntity deleteUser(long id, Jwt jwt) throws IllegalAccessException;
```

Comment:
You have imported a wrong Exception. The one that you should use is under the com.dsec.backend.exception package.

Reply:
I will rename the exception so that there is no conflict anymore.

Reply:
The new name is ForbiddenAccessException